pkg/dataflatten/jwt.go

```diff
@@ -0,0 +1,49 @@
+package dataflatten
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/golang-jwt/jwt/v5"
+)
+
+// JWTFile adds support for the kolide_jwt table, which allows parsing
+// a file containing a JWT. Note that the kolide_jwt table does not handle
+// verification - this is a utility table for convenience.
+func JWTFile(file string, opts ...FlattenOpts) ([]Row, error) {
+	return flattenJWT(file, opts...)
+}
+
+func flattenJWT(path string, opts ...FlattenOpts) ([]Row, error) {
+	results := map[string]interface{}{}
+	jwtFH, err := os.Open(path)
+	if err != nil {
+		return nil, fmt.Errorf("unable to access file: %w", err)
+	}
```

Comment:
I forget how all the layers here work, if you pass the table a path that doesn't exist, will it error out or continue?

Reply:
it will continue, giving zero rows from the missing path. e.g. `where path='/tmp/missing' or path='/tmp/present'` will just return the values found in /tmp/present

Reply:
Cool. Then we handle it up a layer.